src/gen_experiments/utils.py

```diff
@@ -16,6 +16,15 @@
 logger = logging.getLogger(__name__)
 
 
+class PDEData(dict):
```

Comment:
```suggestion
class PDEData(TypedDict):
```
(requires importing `TypedDict` from `typing`